python/cugraph/tests/test_balanced_cut.py

```diff
@@ -120,3 +121,39 @@ def test_digraph_rejected():
 
     with pytest.raises(Exception):
         cugraph_call(G, 2)
+
+
+@pytest.mark.parametrize("graph_file", utils.DATASETS)
+@pytest.mark.parametrize("partitions", PARTITIONS)
+def test_edge_cut_clustering_with_edgevals_nx(graph_file, partitions):
+    gc.collect()
+
+    # Read in the graph and get a cugraph object
+    NM = utils.read_csv_for_nx(graph_file, read_weights_in_sp=True)
+    G = nx.from_pandas_edgelist(
+                NM, create_using=nx.Graph(), source="0", target="1",
+                edge_attr="weight"
+    )
```

Comment:
could you have used `utils.generate_nx_graph_from_file()` here instead?

Reply:
true.  I should update all the test to use that